scripts/src/main/scala/com/gu/mediaservice/scripts/EsScript.scala

```diff
@@ -143,6 +145,43 @@ object UpdateMapping extends EsScript {
   }
 }
 
+object UpdateSettings extends EsScript {
+
+  def run(esHost: String, extraArgs: List[String]) {
+    // TODO: add the ability to update a section of the mapping
+    object EsClient extends ElasticSearchClient {
+      val imagesAlias = "imagesAlias"
+      val port = esPort
+      val host = esHost
+      val cluster = esCluster
+
+      if (esHost != "localhost") {
+        System.err.println(s"You can only run UpdateSettings on localhost, not '$esHost'")
```

Comment:
Can we read the host from the [properties file](https://github.com/guardian/grid/pull/1604)?